spring-xd-dirt/src/main/java/org/springframework/xd/dirt/modules/metadata/JmsSourceModuleOptionsMetadata.java

```diff
@@ -42,6 +44,8 @@
 
 	private String clientId = null;
 
+	private String acknowledge = "auto";
```

Comment:
Wanted to see if we could use enums, but could not because of `dups-ok`.

Then, got me thinking about each combination. Does `dmlc ` only make sense with `transacted`?

```
auto
client
dups-ok
transacted
```

At the very least, I'd like to add jsr303 validation to that option. But not sure about the profiles to activate for each?